setup.py

```diff
@@ -80,7 +80,7 @@ def read(f):
         "GitHub: repo": "https://github.com/aio-libs/multidict",
     },
     license="Apache 2",
-    packages=["multidict"],
+    packages=["multidict", "multidict._multilib"],
     python_requires=">=3.7",
     include_package_data=True,
```

Comment:
I posted a question @ https://github.com/pypa/setuptools/issues/3340#issuecomment-1869926919, but it seems `include_package_data=True` should be re-assessed — do we need it enabled? Why? Would disabling it also remove the warning?

Reply:
I experimented with this and it turned out that removing the setting yields the same distribution contents while not emitting the warning so I've gone with that: https://github.com/aio-libs/multidict/commit/31e11709ad59753f38b5f1a89716788025dc4718.